src/components/CompareResults/beta/ResultsView.tsx

```diff
@@ -31,15 +33,22 @@ function ResultsView(props: ResultsViewProps) {
       backgroundColor: background(themeMode),
     }),
   };
+  const activeComparison: string = useAppSelector(
+    (state: RootState) => state.comparison.activeComparison,
```

Comment:
optional nit: by using ` useAppSelector` I believe you don't need to explicitely type `RootState`, that's exactly the purpose for `useAppSeelctor`.

```suggestion
    (state) => state.comparison.activeComparison,
```